### DISCUSSION TITLE
Frame Removal from an Artwork with a Picture Frame

### DISCUSSION THREAD

**Discussion by jarashanth** [Sanster/IOPaint#450] [Q&A] [answered]
Is that possible to use this tool to remove frames with various shapes (rectangle, circular, oval, etc..) out from a photo/painting frame, automatically?

**Comment by Sanster** [comment]
Can you provide two sample images? It sounds like your requirement is an image segmentation task.

**Comment by jarashanth** [comment]
yeah, it is more like an image segmentation task.

**Reply by Sanster** [reply, marked as answer]
IOPaint is not suitable for this task. You need to find projects that specialize in segmentation tasks, maybe https://github.com/open-mmlab/mmsegmentation.

**Comment by jarashanth** [comment]
![Uploading 0.jpg…]()
